test/jdk/java/nio/channels/DatagramChannel/Connect.java

```diff
@@ -99,36 +110,57 @@ public static class Initiator implements AutoCloseable, Runnable {
 
         public void run() {
             try {
-                ByteBuffer bb = ByteBuffer.allocateDirect(256);
-                bb.put("hello".getBytes());
+                byte[] bytes = MESSAGE.getBytes(US_ASCII);
+                ByteBuffer bb = ByteBuffer.allocateDirect(MAX);
+                bb.put(bytes);
                 bb.flip();
-                log.println("Initiator connecting to " + connectSocketAddress);
+                log.println("Initiator connecting to: " + connectSocketAddress);
                 dc.connect(connectSocketAddress);
+                log.println("Initiator bound to: " + dc.getLocalAddress());
 
                 // Send a message
-                log.println("Initiator attempting to write to Responder at " + connectSocketAddress.toString());
+                log.println("Initiator attempting to write to Responder at " + connectSocketAddress);
                 dc.write(bb);
 
                 // Try to send to some other address
                 try {
                     int port = dc.socket().getLocalPort();
                     InetAddress loopback = InetAddress.getLoopbackAddress();
-                    InetSocketAddress otherAddress = new InetSocketAddress(loopback, (port == 3333 ? 3332 : 3333));
```

Comment:
Hello Daniel,
It was a bit surprising to see a specific port value being used here (before the changes in the PR). So I looked at the history of this test to see if it was intentional and was there for some specific reason. It appears that this was an unintentional leftover from one of the previous refactoring of this test. Using an ephemeral port for this `otherAddress`, like you now do in this PR, looks good to me then.